### PR TITLE
inlyne 0.4.1 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/i/inlyne.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  desc "GPU powered yet browserless tool to help you quickly view markdown files"
+  homepage "https://github.com/Inlyne-Project/inlyne"
+  url "https://github.com/Inlyne-Project/inlyne/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "62e089627812690b182520138708151f47391c5ba518f0b80649942dedd1be87"
+  license "MIT"
+  head "https://github.com/Inlyne-Project/inlyne.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on :macos # currently, the linux build has some issue, notified upstream about it
```
Comment (SMillerDev) [inline Formula/i/inlyne.rb:11]: Please link the issue

File: Formula/i/inlyne.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  uses_from_macos "expect" => :test
```
Comment (SMillerDev) [inline Formula/i/inlyne.rb:13]: Can it not work with the PTY?

Comment (chenrui333) [reply Formula/i/inlyne.rb:13]: yeah, it does not. I tried `w.write "q"` it did not work.

File: Formula/i/inlyne.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/i/inlyne.rb:11]: I shoot a email per their request, I can submit an issue as well though.

Comment (SMillerDev) [reply Formula/i/inlyne.rb:11]: Seems better to keep track of it.

**Review by fxcoudert — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks folks, merging.

**Pull request merged**